src/App.js

```diff
@@ -25,6 +26,7 @@ function App() {
           element={[
             <Header />,
             <SingleArticle />,
+            <CommentList />
           ]}
```

Comment:
If your `Header` component is present on every page, instead of providing it to all of your routes, you can simply render it above your `Routes` component